diff --git i/klippy/extras/display/display.cfg w/klippy/extras/display/display.cfg
index 042c32727..8b3934cb7 100644
--- i/klippy/extras/display/display.cfg
+++ w/klippy/extras/display/display.cfg
@@ -62,7 +62,7 @@ text:
     {% set pos = printer.toolhead.position %}
     { "X%-4.0fY%-4.0fZ%-5.2f" % (pos.x, pos.y, pos.z) }
   {% else %}
-    Ready
+    SiboorV0.2 Ready!
   {% endif %}
 
 
diff --git i/klippy/extras/display/menu.cfg w/klippy/extras/display/menu.cfg
index b0df9fc50..f7714b3ef 100644
--- i/klippy/extras/display/menu.cfg
+++ w/klippy/extras/display/menu.cfg
@@ -14,19 +14,10 @@
 #       + Pause printing
 #       + Resume printing
 #       + Abort printing
-#   + SD Card
-#       + Start printing
-#       + Resume printing
-#       + Pause printing
-#       + Cancel printing
-#       + ... (files)
 #   + Control
 #       + Home All
 #       + Home Z
 #       + Home X/Y
-#       + Z Tilt
-#       + Quad Gantry Lvl
-#       + Bed Mesh
 #       + Steppers off
 #       + Fan: OFF
 #       + Fan speed: 000%
@@ -65,9 +56,7 @@
 #           + Cooldown hotbed
 #   + Filament
 #       + Ex0:000 (0000)
-#       + Load Fil. fast
 #       + Load Fil. slow
-#       + Unload Fil.fast
 #       + Unload Fil.slow
 #       + Feed: 000.0
 #   + Setup
@@ -78,16 +67,6 @@
 #       + PID tuning
 #           + Tune Hotend PID
 #           + Tune Hotbed PID
-#       + Calibration
-#           + Delta cal. auto
-#           + Delta cal. man
-#               + Start probing
-#               + Move Z: 000.00
-#               + Test Z: ++
-#               + Accept
-#               + Abort
-#           + Bed probe
-#       + Dump parameters
 
 ### menu main ###
 [menu __main]
@@ -137,7 +116,7 @@ gcode:
 ### menu octoprint ###
 [menu __main __octoprint]
 type: list
-name: OctoPrint
+name: Print Mode 
 
 [menu __main __octoprint __pause]
 type: command
@@ -160,59 +139,6 @@ name: Abort printing
 gcode:
     {action_respond_info('action:cancel')}
 
-### menu virtual sdcard ###
-[menu __main __sdcard]
-type: vsdlist
-enable: {('virtual_sdcard' in printer)}
-name: SD Card
-
-[menu __main __sdcard __start]
-type: command
-enable: {('virtual_sdcard' in printer) and printer.virtual_sdcard.file_path and not printer.virtual_sdcard.is_active}
-name: Start printing
-gcode: M24
-
-[menu __main __sdcard __resume]
-type: command
-enable: {('virtual_sdcard' in printer) and printer.print_stats.state == "paused"}
-name: Resume printing
-gcode:
-    {% if "pause_resume" in printer %}
-        RESUME
-    {% else %}
-        M24
-    {% endif %}
-
-[menu __main __sdcard __pause]
-type: command
-enable: {('virtual_sdcard' in printer) and printer.print_stats.state == "printing"}
-name: Pause printing
-gcode:
-    {% if "pause_resume" in printer %}
-        PAUSE
-    {% else %}
-        M25
-    {% endif %}
-
-[menu __main __sdcard __cancel]
-type: command
-enable: {('virtual_sdcard' in printer) and (printer.print_stats.state == "printing" or printer.print_stats.state == "paused")}
-name: Cancel printing
-gcode:
-    {% if 'pause_resume' in printer %}
-        CANCEL_PRINT
-    {% else %}
-        M25
-        M27
-        M26 S0
-        TURN_OFF_HEATERS
-        {% if printer.toolhead.position.z <= printer.toolhead.axis_maximum.z - 5 %}
-            G91
-            G0 Z5 F1000
-            G90
-        {% endif %}
-    {% endif %}
-
 ### menu control ###
 [menu __main __control]
 type: list
@@ -236,24 +162,6 @@ enable: {not printer.idle_timeout.state == "Printing"}
 name: Home X/Y
 gcode: G28 X Y
 
-[menu __main __control __z_tilt]
-type: command
-enable: {not printer.idle_timeout.state == "Printing" and ('z_tilt' in printer)}
-name: Z Tilt
-gcode: Z_TILT_ADJUST
-
-[menu __main __control __quad_gantry_level]
-type: command
-enable: {not printer.idle_timeout.state == "Printing" and ('quad_gantry_level' in printer)}
-name: Quad Gantry Lvl
-gcode: QUAD_GANTRY_LEVEL
-
-[menu __main __control __bed_mesh]
-type: command
-enable: {not printer.idle_timeout.state == "Printing" and ('bed_mesh' in printer)}
-name: Bed Mesh
-gcode: BED_MESH_CALIBRATE
-
 [menu __main __control __disable]
 type: command
 name: Steppers off
@@ -553,7 +461,7 @@ type: command
 enable: {('extruder' in printer) and ('heater_bed' in printer)}
 name: Preheat all
 gcode:
-    M140 S110
+    M140 S95
     M104 S245
 
 [menu __main __temp __preheat_abs __hotend]
@@ -566,7 +474,7 @@ gcode: M104 S245
 type: command
 enable: {'heater_bed' in printer}
 name: Preheat hotbed
-gcode: M140 S110
+gcode: M140 S95
 
 [menu __main __temp __cooldown]
 type: list
@@ -608,36 +516,18 @@ input_max: {printer.configfile.config.extruder.max_temp}
 input_step: 1
 gcode: M104 T0 S{'%.0f' % menu.input}
 
-[menu __main __filament __loadf]
-type: command
-name: Load Fil. fast
-gcode:
-    SAVE_GCODE_STATE NAME=__filament__load
-    M83
-    G1 E50 F960
-    RESTORE_GCODE_STATE NAME=__filament__load
-
 [menu __main __filament __loads]
 type: command
-name: Load Fil. slow
+name: Load filament
 gcode:
     SAVE_GCODE_STATE NAME=__filament__load
     M83
     G1 E50 F240
     RESTORE_GCODE_STATE NAME=__filament__load
 
-[menu __main __filament __unloadf]
-type: command
-name: Unload Fil.fast
-gcode:
-    SAVE_GCODE_STATE NAME=__filament__load
-    M83
-    G1 E-50 F960
-    RESTORE_GCODE_STATE NAME=__filament__load
-
 [menu __main __filament __unloads]
 type: command
-name: Unload Fil.slow
+name: Unload filament
 gcode:
     SAVE_GCODE_STATE NAME=__filament__load
     M83
@@ -661,11 +551,6 @@ type: list
 enable: {not printer.idle_timeout.state == "Printing"}
 name: Setup
 
-[menu __main __setup __save_config]
-type: command
-name: Save config
-gcode: SAVE_CONFIG
-
 [menu __main __setup __restart]
 type: list
 name: Restart
@@ -673,15 +558,34 @@ name: Restart
 [menu __main __setup __restart __host_restart]
 type: command
 enable: {not printer.idle_timeout.state == "Printing"}
-name: Restart host
+name: Restart MCU
 gcode: RESTART
 
 [menu __main __setup __restart __firmware_restart]
 type: command
 enable: {not printer.idle_timeout.state == "Printing"}
-name: Restart FW
+name: Restart Klipper
 gcode: FIRMWARE_RESTART
 
+[menu __main __setup __pi-poweroff]
+type: list
+enable: {not printer.idle_timeout.state == "Printing"}
+name: Shutdown Pi
+
+[menu __main __setup __pi-poweroff __no]
+type: command
+name: NO
+enable: {not printer.idle_timeout.state == "Printing"}
+gcode:
+  {menu.back()}
+
+[menu __main __setup __pi-poweroff __yes]
+type: command
+name: YES
+enable: {not printer.idle_timeout.state == "Printing"}
+gcode:
+  SHUTDOWN
+
 [menu __main __setup __tuning]
 type: list
 name: PID tuning
@@ -696,87 +600,9 @@ gcode: PID_CALIBRATE HEATER=extruder TARGET=210 WRITE_FILE=1
 type: command
 enable: {(not printer.idle_timeout.state == "Printing") and ('heater_bed' in printer)}
 name: Tune Hotbed PID
-gcode: PID_CALIBRATE HEATER=heater_bed TARGET=60 WRITE_FILE=1
+gcode: PID_CALIBRATE HEATER=heater_bed TARGET=80 WRITE_FILE=1
 
-[menu __main __setup __calib]
-type: list
-name: Calibration
-
-[menu __main __setup __calib __delta_calib_auto]
+[menu __main __setup __save_config]
 type: command
-enable: {(not printer.idle_timeout.state == "Printing") and ('delta_calibrate' in printer)}
-name: Delta cal. auto
-gcode:
-    G28
-    DELTA_CALIBRATE
-
-[menu __main __setup __calib __delta_calib_man]
-type: list
-enable: {(not printer.idle_timeout.state == "Printing") and ('delta_calibrate' in printer)}
-name: Delta cal. man
-
-[menu __main __setup __calib __bedprobe]
-type: command
-enable: {(not printer.idle_timeout.state == "Printing") and ('probe' in printer)}
-name: Bed probe
-gcode: PROBE
-
-[menu __main __setup __calib __delta_calib_man __start]
-type: command
-name: Start probing
-gcode:
-    G28
-    DELTA_CALIBRATE METHOD=manual
-
-[menu __main __setup __calib __delta_calib_man __move_z]
-type: input
-name: Move Z: {'%03.2f' % menu.input}
-input: {printer.gcode_move.gcode_position.z}
-input_step: 1
-realtime: True
-gcode:
-    {%- if menu.event == 'change' -%}
-        G1 Z{'%.2f' % menu.input}
-    {%- elif menu.event == 'long_click' -%}
-        G1 Z{'%.2f' % menu.input}
-        SAVE_GCODE_STATE NAME=__move__axis
-        G91
-        G1 Z2
-        G1 Z-2
-        RESTORE_GCODE_STATE NAME=__move__axis
-    {%- endif -%}
-
-[menu __main __setup __calib __delta_calib_man __test_z]
-type: input
-name: Test Z: {['++','+','+.01','+.05','+.1','+.5','-.5','-.1','-.05','-.01','-','--'][menu.input|int]}
-input: 6
-input_min: 0
-input_max: 11
-input_step: 1
-gcode:
-    {%- if menu.event == 'long_click' -%}
-        TESTZ Z={['++','+','+.01','+.05','+.1','+.5','-.5','-.1','-.05','-.01','-','--'][menu.input|int]}
-    {%- endif -%}
-
-[menu __main __setup __calib __delta_calib_man __accept]
-type: command
-name: Accept
-gcode: ACCEPT
-
-[menu __main __setup __calib __delta_calib_man __abort]
-type: command
-name: Abort
-gcode: ABORT
-
-[menu __main __setup __dump]
-type: command
-name: Dump parameters
-gcode:
-   {% for name1 in printer %}
-      {% for name2 in printer[name1] %}
-         { action_respond_info("printer['%s'].%s = %s"
-                               % (name1, name2, printer[name1][name2])) }
-      {% else %}
-         { action_respond_info("printer['%s'] = %s" % (name1, printer[name1])) }
-      {% endfor %}
-   {% endfor %}
+name: Save config
+gcode: SAVE_CONFIG
\ No newline at end of file
